lib/logstash/outputs/elasticsearch/http_client/manticore_adapter.rb

```diff
@@ -0,0 +1,57 @@
+require 'manticore'
+
+module LogStash; module Outputs; class ElasticSearch; class HttpClient;
+  class ManticoreAdapter
+    attr_reader :manticore, :logger
+
+    def initialize(logger, options)
+      @logger = logger
+      build_client(options || {})
+    end
+
+    # Should just be run once at startup
+    def build_client(options={})
```

Comment:
it should be more explicit that this is a private api, called only by initialize